### ISSUE TITLE
Broken versions >= 0.6.8 <= 0.6.12

### ISSUE THREAD

**Issue by jweyrich** [ar90n/serverless-s3-local#153]
serverless-s3-local is failing on

```
const { default: Lambda } = require('serverless-offline/dist/lambda/'); 
```

Stack trace:

>   Error: Cannot find module 'serverless-offline/dist/lambda/'
      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
      at Function.Module._load (internal/modules/cjs/loader.js:562:25)
      at Module.require (internal/modules/cjs/loader.js:692:17)
      at require (internal/modules/cjs/helpers.js:25:18)
      at Object.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/serverless-s3-local/index.js:8:29)
      at Module._compile (internal/modules/cjs/loader.js:778:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
      at Module.load (internal/modules/cjs/loader.js:653:32)
      at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
      at Function.Module._load (internal/modules/cjs/loader.js:585:3)
      at Module.require (internal/modules/cjs/loader.js:692:17)
      at require (internal/modules/cjs/helpers.js:25:18)
      at requireServicePlugin (/opt/atlassian/pipelines/agent/build/node_modules/serverless/lib/classes/PluginManager.js:27:12)
      at pluginsObject.modules.filter.map.name (/opt/atlassian/pipelines/agent/build/node_modules/serverless/lib/classes/PluginManager.js:144:20)

Some package.json dev dependencies we have:

> "devDependencies": {
    // ... removed various unrelated deps.
    "@types/aws-lambda": "^8.10.31",
    "@types/aws-sdk": "^2.7.0",
    "@types/node": "^12.6.8",
    "nodemon": "^1.19.4",
    "serverless": "^1.83.2",
    "serverless-offline-ssm": "^5.0.1",
    "serverless-dotenv-plugin": "^3.1.0",
    "serverless-dynamodb-local": "^0.2.39",
    "serverless-jetpack": "^0.10.7",
    "serverless-offline": "^5.12.1",
    "serverless-offline-sqs": "^3.0.0",
    "serverless-plugin-stage-variables": "^1.9.5",
    "serverless-pseudo-parameters": "^2.5.0",
    "serverless-s3-local": "^0.6.7",
    "typescript": "^4.0.5"
  },

Note we have ^0.6.7 which is now fetching the broken 0.6.12.

**Comment by ar90n** [comment]
@jweyrich 
Thanks for your great report! I reproduced an issue you said.
I fixed some dependencies and publish a new version. Please try v0.6.13.
If you meet the same issue in the new version, please let me know.
I want to fix this issue correctly.

**Comment by jweyrich** [comment]
Sure. Will try the new version tomorrow and will let you know.

**Comment by ar90n** [comment]
I fix the dependency issue again and I published v0.6.14. Please use it!

**Comment by jweyrich** [comment]
The new version fixed it. Thank you!
